commands/project/upload.ts

```diff
@@ -27,19 +27,19 @@ const { EXIT_CODES } = require('../../lib/enums/exitCodes');
 
 const i18nKey = 'commands.project.subcommands.upload';
 
-exports.command = 'upload [path] [--forceCreate] [--message]';
+exports.command = 'upload';
 exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
 
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
-  const { forceCreate, path: projectPath, message, derivedAccountId } = options;
+  const { forceCreate, message, derivedAccountId } = options;
   const accountConfig = getAccountConfig(derivedAccountId);
   const accountType = accountConfig && accountConfig.accountType;
 
   trackCommandUsage('project-upload', { type: accountType }, derivedAccountId);
 
-  const { projectConfig, projectDir } = await getProjectConfig(projectPath);
+  const { projectConfig, projectDir } = await getProjectConfig();
```

Comment:
Does this mean that the user needs to run the command from within a project folder? Should we add some help text to indicate that?

Reply:
Yeah I can look into making this more obvious via help text

Reply:
Added better copy in [6d99f67](https://github.com/HubSpot/hubspot-cli/pull/1266/commits/6d99f67a0643d7d87a8c75fc8f866891f70fcfeb)